src/EA.Weee.Web/Areas/Scheme/Controllers/OutgoingTransfersController.cs

```diff
@@ -104,6 +104,11 @@ public async Task<ActionResult> SubmittedTransfer(Guid pcsId, Guid evidenceNoteI
             {
                 var noteData = await client.SendAsync(User.GetAccessToken(), new GetTransferEvidenceNoteForSchemeRequest(evidenceNoteId));
 
+                if (noteData.Status != NoteStatus.Submitted)
```

Comment:
here too, that's an action link for the submitted transfers only no?

Reply:
Yeah this only applies to the review screen and is to prevent the user accessing the screen again when its not in submitted status